src/scss/typography.scss

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Montserrat&display=swap");
```

Comment:
Is there a reason these two lines are also in `main.scss` or is that a typo?

Reply:
Typo, will remove!